HomeTask3/calculateWithFunctions.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * 
+ * @param {number} num the number passed to return calculated value
+ * @param {function} func the function that needs to executed with num default value returns number
+ * 
+ * @throws {error} if the return number accepts number function again like zero
+ */
+const returnNumber = (num, func = (number) => number) => {
+    if (typeof func != "function") {
+        throw new Error("number can't process a number")
```

Comment:
What is number can't process a number? Please rephrase it

Reply:
added "Operator not found between numbers" as Error message